### ISSUE TITLE
usually nighlty macos are around 57mb, sometime around 37 and they don't open

### ISSUE THREAD

**Issue by wnc2** [supermerill/SuperSlicer#2757]
the issue happens often.

last (not working) version:
![Schermata 2022-04-26 alle 10 07 54](https://user-images.githubusercontent.com/43959480/165253108-b55251a2-db52-4e67-bf1d-512356b64aa6.jpg)

last but one (working) version:
![Schermata 2022-04-26 alle 10 07 23](https://user-images.githubusercontent.com/43959480/165253338-44ed0efa-cf41-4567-981f-32441cb04ded.jpg)

**Comment by supermerill** [comment]
yes. The macos build has a problem: when it fails, it doesn't show itself as failed but and publish a dmg without the software inside.

**Comment by wnc2** [comment]
latest is still bugged, why you publish them if they are bugged?
![Schermata 2022-04-30 alle 09 56 10](https://user-images.githubusercontent.com/43959480/166097175-afcfc12c-a835-49a7-9668-1ee39f3e82de.jpg)

**Comment by supermerill** [comment]
> latest is still bugged, why you publish them if they are bugged?

The macos build has a problem: when it fails, it doesn't show itself as failed but and publish a dmg without the software inside.
I don't publish these, it's just the output of the automated builds.
What I publish is in the releases page.
